README.md

```diff
@@ -24,7 +24,7 @@ When running `make init`, you may encounter an error like this:
     no matching version
 ```
 
-To address this, first run `opam update`, then rerun `make init`.
+Which means `opam` doesn't have the latest packages definitions from the opam-repository. To address this, first run `opam update`, then rerun `make init`.
```

Comment:
I'm also not sure how much this conflict message is up-to-date now that we've release 2.0 too...

```suggestion
It means `opam` doesn't have the latest packages definitions from the opam-repository. To address this, first run `opam update`, then rerun `make init`.
```